tests/core/conftest.py

```diff
@@ -84,6 +84,13 @@ def __init__(
         super(MongoTrackerStore, self).__init__(_domain, None)
 
 
+@pytest.yield_fixture(scope="session")
+def event_loop(request):
```

Comment:
why is this fixture necessary? it's not passed as an argument into any test function

Reply:
The reason why it's needed described here: https://github.com/pytest-dev/pytest-asyncio/issues/68

Long story short: this `event_loop` is used by `pytest-asyncio`, and redefining it is currently the only supported way of changing the scope of this fixture. So basically the only line that makes real sense here is this one:
```
@pytest.yield_fixture(scope="session")
```

Changing the scope of this function is required to avoid ScopeMismatch errors like this one:
```
ScopeMismatch: You tried to access the 'function' scoped fixture 'event_loop' with a 'session' scoped request object, involved factories
```

Reply:
I should actually comment it there.